modules/creating-instance-aws-load-balancer-controller.adoc

```diff
@@ -0,0 +1,125 @@
+// Module included in the following assemblies:
+//
+// * networking/aws_load_balancer_operator/create-instance-aws-load-balancer-controller.adoc
+
+:_content-type: PROCEDURE
+[id="nw-creating-instance-aws-load-balancer-controller_{context}"]
+= Creating an instance of the AWS Load Balancer Controller using ALB Operator
+
+After installing the operator, you can create an instance of the `AWSLoadBalancerController`. You can install only a single instance of the `aws-load-balancer-controller` in a cluster. The operator reconciles only the resource with the name `cluster`.
+
+.Prerequisites
```

Comment:
Additional prerequisites are needed. I've added the needed docs [here](https://github.com/openshift/aws-load-balancer-operator/pull/61), You pull them in once that PR is merged.